README.md

```diff
@@ -16,19 +16,49 @@
 --->
 
 <p align="center">
-  <img src="https://github.com/parallaxsecond/parsec/blob/master/parsec.png?raw=true" alt="Parsec logo"/>
```

Comment:
Should we delete the old logo `parsec.png` in this PR as well?

Reply:
Good point, I didn't think about that. However, there will be a follow-up change coming very soon where I add in the partner logos so I can remove that one at the same time. Also, once this PR is merged, we need to update the logo on the GitHub org page. I have a square-format shield logo (without the PARSEC text) that I can use for that.

Reply:
Ok, no worries then!